### ISSUE TITLE
ElvUI ChatBoxes

### ISSUE THREAD

**Issue by Macumbafeh** [laytya/WIM-3.0#1]
Hello,

First of all thanks for this fixed version.
Someone reported me this issue with your version of WIM.
Your version fix all the bug the tbc version had but there is no support for the ElvUI chat frames.

It bugs out elvui chatboxes and makes them like default blizzard chatboxes instead of being handled by elvui.

Is it possible to fix it please? Since the tbc version has this feature.
This version : http://www.mediafire.com/file/up73x5cbpwb711k/WIM-2.4.15.zip/file

Thanks !

**Comment by laytya** [comment]
may be its elvui needs to make compatible to WIM not WIM =)))

**Comment by Macumbafeh** [comment]
Yeah i was thinking about it after, i could be a elvUI issue instead of WIM :)
Strange though it works with the 2.4.15 version and not the 3.0

Thnaks for the reply

**Comment by laytya** [comment]
i think elvui tested w/ 2.4 but 3.0 is from WTOLK

**Comment by Macumbafeh** [comment]
If i may, i have this only on ElvUI's chat module related to WiM only

![image](https://user-images.githubusercontent.com/47739411/233545190-e12f735a-bc72-431e-83d8-690071a9fc3e.png)

### Text format:
   if WIM then
      WIM.RegisterWidgetTrigger("chat_display", "whisper,chat,w2w,demo", "OnHyperlinkClick", function(self) CH.clickedframe = self end)
      WIM.RegisterItemRefHandler("url", HyperLinkedURL)
   end

Do i need to add if IsAddOnLoaded("WIM") then?
I see no local...

The only difference i can see is on wim.toc
2.4.15
![image](https://user-images.githubusercontent.com/47739411/233546470-66de91e1-278a-40ee-8df6-8e7bbd155ed6.png)

3.0
![image](https://user-images.githubusercontent.com/47739411/233546524-5eac7ca6-9dd9-485e-8b5f-240ec98bc845.png)

Only the SavedVariables change, maybe it's that?
